### DISCUSSION TITLE
Ability to make certain commands cost XP levels.

### DISCUSSION THREAD

**Discussion by PieCreeper12** [EssentialsX/Essentials#5579] [Ideas and Feature Suggestions]
My suggestion is the ability to make commands of your choice use a configurable amount of XP when used. For example, I could make it so if a player does /home or /tpa and they teleport, it lowers their XP level by 1. This would encourage players to travel manually rather than always teleporting. Commands that cost XP will not lower XP for players in creative mode.

**Option:** Does the player need enough XP to use the command? If false, it will use whatever XP the player has. If true, the player needs as much XP the command uses in order to use it.

**Comment by Evidentsinger14** [comment]
This seems kinda niche. In my opinion this should instead be implemented as a vault supported economy plugin and using that on top of Essentials.
